test/db/index.js

```diff
@@ -0,0 +1,86 @@
+const should = require('chai').should();
+const fs = require('fs-extra');
+const { initDB, db } = require('../../src/db');
+const { getDbDir } = require('../../src/config');
+const DBHelper = require('../../src/db/db-helper');
+
+describe('db', () => {
+  before(async () => {
+    await initDB();
+  });
+
+  describe('test block db', () => {
+    describe('insert', () => {
+      let blockCount;
+      const toBeInsertedBlockNum = 11111;
+      const toBeInsertedBlockTime = Date.now();
+      it('should start with empty block db', async () => {
+        blockCount = await db.Blocks.count({});
+        blockCount.should.equal(0);
+      });
+
+      it('should insert with new block num', async () => {
+        await DBHelper.insertBlock(toBeInsertedBlockNum, toBeInsertedBlockTime);
+        blocks = await db.Blocks.find({});
+        blocks.length.should.equal(1);
+        blocks[0].blockNum.should.equal(toBeInsertedBlockNum);
+        blocks[0].blockTime.should.equal(toBeInsertedBlockTime);
+      });
+
+      it('should not insert with existed block num', async () => {
+        const toBeInsertedFailBlockTime = Date.now();
+        await DBHelper.insertBlock(toBeInsertedBlockNum, toBeInsertedFailBlockTime);
```

Comment:
this should throw here so you need to catch the throw and assert there is an error

Reply:
but in the DBHelper.insertBlock, it will check if it is existed, it it's already there, the method just return, no error throwed in that method.